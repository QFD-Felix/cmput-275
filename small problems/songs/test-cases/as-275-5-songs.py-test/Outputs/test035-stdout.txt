34 15 1 3 16 8 3 28 2 14 8 0 0 1 5 11 1 8 10 2 4 8 0 4 7 8 35 12 2 12 20 5 0 5 8 8 8 33 16 1 15 7 5 9 1 1 16 2 5 24 24 0 5 3 11 8 20 1 37 0 0 8 2 5 2 2 12 32 15 2 2 33 8 28 1 4 1 5 6 8 1 7 26 7 10 0 0 15 1 2 2 12 1 20 2 5 2 0 24 12 27 6 3 15 8 1 3 27 44 1 6 8 2 2 7 14 1 7 19 1 16 20 1 5 2 0 20 4 2 9 2 4 1 12 5 12 12 3 38 20 4 14 0 0 10 16 5 3 1 39 14 5 14 0 3 10 4 1 14 8 2 7 13 0 45 4 5 3 16 10 4 6 15 0 12 15 4 1 14 2 2 12 3 0 16 9 3 4 7 23 8 20 4 6 3 19 9 10 4 2 4 5 1 7 31 17 2 6 7 11 18 5 5 16 9 9 15 4 0 0 20 9 5 3 20 0 11 11 5 3 24 8 24 9 0 10 0 9 3 22 20 9 11 0 15 4 2 0 9 26 1 16 0 21 1 8 13 1 4 17 11 23 2 14 1 5 0 1 4 22 47 10 14 0 15 3 7 13 4 0 0 47 10 2 12 3 1 11 16 20 3 10 7 2 0 44 15 0 2 3 3 4 5 3 26 7 2 6 8 1 22 10 3 11 0 3 4 6 9 0 2 8 13 20 1 23 0 7 3 4 10 5 10 4 26 12 41 13 0 2 12 3 21 3 1 9 15 4 1 4 9 14 15 5 1 39 0 0 17 9 6 3 0 10 29 17 10 5 3 0 0 3 15 8 5 5 2 8 17 7 3 7 8 0 0 0 0 0 4 0 1 0 0 0 0 0 0
