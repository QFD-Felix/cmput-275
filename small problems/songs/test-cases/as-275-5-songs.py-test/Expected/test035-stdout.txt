35 15 1 3 16 8 3 29 2 14 8 0 0 1 5 11 1 8 10 2 4 8 0 4 8 9 36 13 3 1 21 11 5 0 5 8 8 8 33 16 1 15 7 5 9 1 1 16 2 5 24 24 0 5 3 11 8 20 1 37 0 0 8 2 5 2 2 12 32 15 2 2 33 8 28 1 4 1 5 6 8 1 7 26 7 10 0 0 15 1 2 2 12 1 20 2 5 2 0 24 12 27 6 3 15 8 1 3 27 44 1 6 8 2 2 7 14 1 7 19 1 16 20 1 5 2 0 20 4 2 9 2 4 1 12 5 12 12 3 42 20 4 14 0 0 10 16 5 3 1 13 17 5 18 0 3 8 4 1 18 6 4 11 17 29 5 1 5 1 5 44 5 3 16 10 4 6 15 0 12 15 4 1 14 2 2 12 3 0 16 9 3 4 7 23 8 20 4 6 3 19 9 10 4 2 4 5 1 7 31 17 2 6 7 11 18 5 5 16 9 9 15 4 0 0 20 9 5 3 20 0 11 11 5 3 24 8 24 9 0 10 0 9 3 22 20 9 11 0 15 4 2 0 9 26 1 16 0 21 1 8 13 1 4 17 11 25 2 14 1 5 0 1 4 25 50 10 17 0 13 3 7 8 4 0 0 50 13 1 15 3 7 4 4 1 11 16 20 3 10 7 2 0 44 15 0 2 3 3 4 5 3 26 7 2 6 8 1 22 10 3 11 0 3 4 6 9 0 2 8 13 20 1 23 0 7 3 4 10 5 10 4 26 12 41 13 0 2 12 3 21 3 1 9 15 4 1 4 9 14 15 5 1 39 0 0 17 9 6 3 0 10 29 17 10 5 3 0 0 3 15 8 5 5 2 8 17 7 3 7 8 0 0 0 0 0 4 0 1 0 0 0 0 0 0
